helpers.js

```diff
@@ -36,3 +36,16 @@ export const formatDateDiff = (recent, old) => {
     day: "numeric",
   }).format(old.getTime()); // ex: Nov 15
 };
+
+export function formatLargeNumber(n) {
+  let s = "";
+  if (n < 1000) {
+    s = n;
+  } else if (n >= 1000 && n < 1000000) {
+    s = (Math.round(n)/1000).toFixed(1) + "k"
+  } else if (n >= 1000000) {
+    s = (Math.round(n)/1000000).toFixed(1) + "M"
```

Comment:
Same problem as above, only with other numbers > 1,000,000.

Reply:
Using `Math.trunc` instead of `Math.round` e.g. `(Math.trunc(n)/1000) + "k"` resolves the issue of 500.0k to 500k.  However, I'm seeing how I can alter this to make 499999 = 499.9k instead of 499.999k

Reply:
You may try to convert the result to string first, split it by '.' character and add some if else you statement whether or not you want to display the decimal place

Reply:
I think you're limiting yourself with the use of built-in methods and trying too much to integrate them, that when they don't provide the exact solution you need, you get burnt and don't know the way out.. and I understand.

What if you tried using simple techniques and array operations instead which I believe you can as you worked through some challenges on Codewars.

I'll give you a hint

```suggestion
    let temp = String(n/1000000).split(".");
    // add some conditional operations bellow
    // try it without accessing indices out of the array's range
    s += "M";
```